com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.RenderGraph.cs

```diff
@@ -860,7 +860,13 @@ TextureHandle ContrastAdaptiveSharpeningPass(RenderGraph renderGraph, HDCamera h
                 {
                     passData.source = builder.ReadTexture(source);
                     passData.parameters = PrepareContrastAdaptiveSharpeningParameters(hdCamera);
-                    TextureHandle dest = GetPostprocessOutputHandle(renderGraph, "Contrast Adaptive Sharpen Destination");
+                    TextureHandle dest = renderGraph.CreateTexture(new TextureDesc(Vector2.one, false, true)
```

Comment:
I am wondering if at this point we should allocate this texture with explicit texture size rather than a scale of 1 ( matching the _OutputTextureDimensions passed to CAS )

Reply:
hmm maybe. Gotta test with Software resolution, and ensuring that in dx11 we dont reallocate a texture.

Reply:
On second thought it is indeed probably not a good idea, let's keep like this